docs/sdk.md

```diff
@@ -71,11 +76,12 @@ const path = require( 'path' );
 exports.config = ( {
 	{ argv: { outputDir },
 	getBaseConfig,
+	__rootDir,
```

Comment:
can we drop the __ prefix? those usually have special meaning (provided by node) and in this case it's just a normal variable.

Reply:
it's normal but the reason I added it was to mirror `__dirname` to make some symmetry there. the node built-in path variables have this: `__dirname` and `__filename`

thoughts?

Reply:
Probably good to drop `__` so that folks don't go searching it from Node.js docs? Better to do that in a separate PR though.

Reply:
Maybe `calypsoRoot` for clarity? It's there specifically to avoid resolving to `cwd` which might be different than Calypso root directory.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/27067